pkgs/misc/deepspeech/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv, lib, fetchurl, autoPatchelfHook }:
+
+stdenv.mkDerivation rec {
+  name = "deepspeech-${version}";
+  version = "0.9.3";
+
+  src = fetchurl {
+    url = "https://github.com/mozilla/DeepSpeech/releases/download/v${version}/native_client.amd64.cpu.linux.tar.xz";
+    sha256 = "1qy2gspprcxi76jk06ljp028xl0wkk1m3mqaxyf5qbhhfbvvpfap";
+  };
+  setSourceRoot = "sourceRoot=`pwd`";
```

Comment:
```suggestion
```
that shouldn't be required.

Reply:
Indeed, but otherwise the archive does not get unpacked, I took this from another package in `nixpkgs`